src/components/BingoCard.tsx

```diff
@@ -64,34 +65,37 @@ const BingoCard = ({ cardId }) => {
   };
 
   //Takes a screenshot of the card and downloads it
-  const copyToClipboard = () => takeScreenShot(ref.current).then(download);
+  const downloadScreenshot = () => takeScreenShot(ref.current).then(download);
 
   return (
     <div>
-      <div class="bingo-card-wrapper">
+      <div className="bingo-card-wrapper">
         <div className="bingo-card" ref={ref}>
-        {card.map((row, rowIndex) => (
-          <div key={rowIndex} className="bingo-row">
-            {row.map((num, colIndex) => (
-              <div
-                key={colIndex}
-                className={`bingo-cell ${
-                  marked[rowIndex][colIndex] ? "marked" : ""
-                } ${num ? "" : "empty"}`}
-                onClick={() => {
-                  if (num) toggleNumber(rowIndex, colIndex);
-                }}
-              >
-                {num || ""}
-              </div>
-            ))}
-          </div>
-        ))}
+          {
+            marked?.length ?
```

Comment:
Since we are initialising the marked state in the mounting hook we can get an exception iterating over marked fore init. Preferrably the init should also be readied in a parent component so this component doesn't have to worry what is ready and what isn't.